cjwkernel/types.py

```diff
@@ -615,6 +615,21 @@ def TODO_i18n(cls, text: str) -> I18nMessage:
         """
         return cls("TODO_i18n", {"text": text})
 
+    @classmethod
+    def trans(
+        cls,
+        message_id: str,
+        *,
+        default: str,
+        parameters: Dict[str, Union[int, float, str]] = {},
+    ) -> I18nMessage:
+        """
+        Build an I18nMessage.
+
```

Comment:
Please add a comment explaining why people should use this. My suggestion:

```
Use this method (instead of constructing `I18nMessage` directly) when you wish to mark a string for translation. Workbench's tooling will extract messages from all `I18nMessage.trans()` calls and send them to translators.
```